geode-core/src/main/java/org/apache/geode/management/internal/cli/LogWrapper.java

```diff
@@ -45,10 +45,9 @@
 
   private Logger logger;
 
-  private LogWrapper() {
+  private LogWrapper(Cache cache) {
```

Comment:
I am not sure why LogWrapper needs a cache instance. LogWrapper is used by gfsh logging. And on server/locators, we are solely using LogService now. Probably check with Kirk and see if we can get rid of the the usage of cache inside LogWrapper.

Reply:
@kirklund Any insight?  I looks like we're grabbing the cache's log handler so that things appear in the cache's logs?  I would imagine there are places where the `LogWrapper` can't be replaced by the `LogService.getLogger()`, or else it would've been part of that initial rework.

Reply:
@metatype Someone mentioned elsewhere that you might have insight on how the `java.util` loggers and the cache's handlers interact?

Reply:
Maybe we get delete LogWrapper and change GFSH code to use Log4J 2? You might want to try that in a branch and run as many tests as you can. Just change any GFSH class that uses JULI to instead get a Logger from LogService.